azerothcore/azerothcore-wotlk
# fix(CORE/Commands): Renamed .character deleted old to .character deleted purge

## CHANGES PROPOSED:
-  Renamed .character deleted old [keepDays] to .character deleted purge [keepDays]
-  Updated command description 


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2925



## TESTS PERFORMED:
1. Changed CharDelete.Method to 1 and CharDelete.KeepDays to > 0
2. Created and deleted character
3. Checked DB for the deleted character.
4. Executed .character deleted purge 0
5. Checked if deleted character is gone from the DB



## HOW TO TEST THE CHANGES:

1. Change CharDelete.Method to 1 and CharDelete.KeepDays to > 0
2. Create and delete a character
3. Check DB for the deleted character.
4. Execute .character deleted purge 0
5. Check if deleted character is gone from the DB

## KNOWN ISSUES AND TODO LIST:

- [ ] Update WIKI


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
